helm/values.yaml

```diff
@@ -11,3 +11,10 @@ timecard:
     name: callisto-timecard-restapi
     port: 3000
   host: timecard.dev.callisto.homeoffice.gov.uk
+
+accruals:
+  name: callisto-accruals-restapi
+  service:
+    name: callisto-accruals-restapi
+    port: 3000
```

Comment:
Should this port be the same as the above timecard api?

Reply:
I think because they are different services the ports don't matter, although it is a bit confusing when you see them using the same port. It might be clearer if we changed both services to use different ports